xdem/coreg.py

```diff
@@ -1084,7 +1086,7 @@ def coregister(reference_raster: Union[str, gu.georaster.Raster], to_be_aligned_
     return aligned_raster, error
 
 
-def _transform_to_bounds_and_res(shape: tuple[int, int],
+def _transform_to_bounds_and_res(shape: tuple[int, ...],
```

Comment:
What do the '...' mean?

Reply:
It means the tuple can have an arbitrary length. The shape of an ndarray can't be known by the linter before running, so it needs to be arbitrary.

The linters are also stupid the way that you can't write:
```python
dem_shape: tuple[int, int] = dem.shape[:2]
```
Slicing doesn't work, even though it really should...

Although you could write:
```python
dem_shape: tuple[int, int] = (dem.shape[0], dem.shape[1])
```
but at that point, it just felt so stupid that I reverted to `tuple[int, ...]` as the type...